crates/integration_tests/Cargo.toml

```diff
@@ -34,5 +34,6 @@ iceberg-catalog-rest = { workspace = true }
 iceberg-datafusion = { workspace = true }
 iceberg_test_utils = { path = "../test_utils", features = ["tests"] }
 parquet = { workspace = true }
+serial_test = "*"
```

Comment:
Hi, it's better to have a clear version.

Reply:
Agreed, though with #924 merged this is no longer needed, so I removed it now.